cfpb/rental-assistance-finder
# Fetch external program JSON

Fetch programs data from external JSON file

## Additions
- Utility function to fetch JSON file from s3
- UseEffect hook to load data when App component mounts
- IE11 polyfills
- Error handling and data error message
- Loading notification

## Changes
- Update code to handle new data format
- Move tool into new component


## Testing

1. Check out branch and run yarn start
2. Verify that data is fetched from remote JSON file and loads as expected
3. To test error handling, change program data URL and verify that error message is displayed
